TicTacToeProject/HumanPlayer.cs

```diff
@@ -58,7 +34,7 @@ public bool isValidSpace(string location)
             try
             {
                 var convertedLocation = Convert.ToInt32(location);
-                return _board.getAvailableSpaces().Contains(convertedLocation);
+                return Board.getAvailableSpaces().Contains(convertedLocation);
```

Comment:
`Board` only being necessary in `getMove()` is a sign that it shouldn't be an instance variable. It's strange to see the board get passed to the Player constructor because you expect the board to change over the course of the game. It may work because the board is passed by reference and so the underlying data is mutated but it's not intuitive to think about. It'd be better if you passed the board to the `getMove(board)` method.